bin/install-deps.sh

```diff
@@ -20,7 +20,7 @@ POSTGRES_EXPORTER_VERSION=0.7.0
 PGMONITOR_COMMIT='v3.2'
 OPENSHIFT_CLIENT='https://github.com/openshift/origin/releases/download/v3.11.0/openshift-origin-client-tools-v3.11.0-0cbc58b-linux-64bit.tar.gz'
 CERTSTRAP_VERSION=1.1.1
-YQ_VERSION=2.4.0
+YQ_VERSION=3.3.0
```

Comment:
I would make the version bump of `yq` a separate commit. One that comes before the changes you present.

Reply:
The `yq` version change is now in its own commit.